jest.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-    projects: []
+    projects: ['<rootDir>/libs/dotcms']
```

Comment:
what is this for ? why just one lib ?

Reply:
Is the only lib we have jest tests for. the app uses karma.